external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSourceProviderSuite.scala

```diff
@@ -30,52 +30,100 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 class KafkaSourceProviderSuite extends SparkFunSuite with PrivateMethodTester {
 
+  private val expected = "666"
```

Comment:
Ur, could you pick up another number? This value has some religious meaning. :)

Reply:
Ah, didn't realize it just hit the keyboard and crafted a number which is rarely default value. Thanks for notifying it was not intentional :)